### PR TITLE
Harbour.Harbour version 3.0.0

### PR DESCRIPTION
**Pull request by sarvex**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230423112824

- Is there an installer switch to install automatically?

**Comment by sarvex** [comment]
@stephengillie `/S` seems to be working efficiently, but I am unable to find a way to to edit the PR. Kindly help me with the same

File: manifests/h/Harbour/Harbour/3.0.0/Harbour.Harbour.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Harbour.Harbour
+PackageVersion: 3.0.0
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://sourceforge.net/projects/harbour-project/files/binaries-windows/3.0.0/harbour-3.0.0-win.exe/download
+  InstallerSha256: DCEE5BE89D47A0C85A196483472B5B3EC78C8EE971B7F4AB1F902F273112E51F
+  InstallerSwitches:
+    Silent: /VERYSILENT
```
Comment (stephengillie) [inline manifests/h/Harbour/Harbour/3.0.0/Harbour.Harbour.installer.yaml:12]:
```suggestion
    Silent: /s
```

**Comment by stephengillie** [comment]
> @stephengillie `/S` seems to be working efficiently, but I am unable to find a way to to edit the PR. Kindly help me with the same

GitHub's Files Changed page has very many features. I'm only aware of some of them. Click the blue plus to edit one line:
image

Or click-and-drag the blue plus to edit multiple lines:
image

Use the "suggestion" button to have the lines show up for editing:
image

And use the Preview tab to verify, before clicking Single Comment (or starting a Review, I'm unsure what that entails.)
image

**Comment by sarvex** [comment]
@stephengillie Thanks a lot for the detailed instructions

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230502023534

File: manifests/h/Harbour/Harbour/3.0.0/Harbour.Harbour.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Harbour.Harbour
+PackageVersion: 3.0.0
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://sourceforge.net/projects/harbour-project/files/binaries-windows/3.0.0/harbour-3.0.0-win.exe/download
+  InstallerSha256: DCEE5BE89D47A0C85A196483472B5B3EC78C8EE971B7F4AB1F902F273112E51F
+  InstallerSwitches:
+    Silent: /s
+    SilentWithProgress: /SILENT
```
Comment (stephengillie) [inline manifests/h/Harbour/Harbour/3.0.0/Harbour.Harbour.installer.yaml:13]:
```suggestion
    Silent: /S
    SilentWithProgress: /S
```

File: manifests/h/Harbour/Harbour/3.0.0/Harbour.Harbour.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Harbour.Harbour
+PackageVersion: 3.0.0
+PackageLocale: en-US
+Publisher: Harbour
+PackageName: Harbour
```
Comment (stephengillie) [inline manifests/h/Harbour/Harbour/3.0.0/Harbour.Harbour.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://harbour.github.io/
PrivacyUrl:   https://harbour.github.io/terms
PackageName: Harbour
PackageUrl: https://sourceforge.net/projects/harbour-project/files/
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**